app/assets/stylesheets/common/admin/customize.scss

```diff
@@ -923,3 +923,80 @@ table.permalinks {
     }
   }
 }
+
+.form-templates {
+  &--info {
+    margin-top: 1rem;
+  }
+
+  &--table {
+    .admin-list-item .action {
+      text-align: right;
+    }
+  }
+
+  .btn-primary {
+    margin-top: 1rem;
+  }
```

Comment:
A little bit unexpected here, maybe would deserve more precise selector

Reply:
Yeah,. maybe move the margin to the table element just above.